app/src/views/features/sessions/SessionViewer/SavedSessionViewer.tsx

```diff
@@ -119,9 +119,16 @@ const SavedSessionViewer: React.FC = () => {
         setIsFetching(false);
       })
       .catch((err) => {
-        err.name === "PermissionDenied" && setShowPermissionError(true);
-
-        err.name === "NotFound" && setShowNotFoundError(true);
+        switch (err.name) {
+          case "PermissionDenied":
+            setShowPermissionError(true);
+            break;
+          case "NotFound":
+            setShowNotFoundError(true);
```

Comment:
minor - Moving `NotFound` case one level up, you won't need separate case for `PermissionDenied`.